Additional instructions.

Jehreg opened the pull request:
Simple additional instructions.

Jehreg commented:
I have a more complete pull request coming.

km4ack commented:
I prefer pointing users to https://www.raspberrypi.org/downloads/ for their images. Is there a reason why we wouldn't want to do that?

Jehreg commented:
Sorry, I closed this pull request.  I opened pull request #113 with more stuff in it.

Your question is still relevant to pull #113.  I added instructions on getting arm64 download instructions as I believe that Raspbian will ultimately replace armhf Raspbian on Pi4.  But I added the note that this breaks VNC server.

I have no problems with dropping that paragraph, but "Power users" like me will run into this problem if they grab arm64, so I thought it was worth mentioning.  Perhaps at the bottom of the README then ?
